src/main/java/org/alliancegenome/curation_api/model/entities/Reference.java

```diff
@@ -16,27 +13,56 @@
 import org.hibernate.search.mapper.pojo.mapping.definition.annotation.IndexedEmbedded;
 import org.hibernate.search.mapper.pojo.mapping.definition.annotation.IndexingDependency;
 
-import com.fasterxml.jackson.annotation.JsonView;
-
-import lombok.Data;
-import lombok.EqualsAndHashCode;
-import lombok.ToString;
+import javax.persistence.*;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
 
 @Audited
 @Entity
 @Data
 @EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
 @ToString(callSuper = true)
 @Schema(name = "Reference", description = "POJO that represents the Reference")
-@AGRCurationSchemaVersion(min = "1.4.0", max = LinkMLSchemaConstants.LATEST_RELEASE, dependencies = { InformationContentEntity.class }, partial = true)
+@AGRCurationSchemaVersion(min = "1.4.0", max = LinkMLSchemaConstants.LATEST_RELEASE, dependencies = {InformationContentEntity.class}, partial = true)
 public class Reference extends InformationContentEntity {
 
 	@IndexedEmbedded(includeDepth = 1)
 	@IndexingDependency(reindexOnUpdate = ReindexOnUpdate.SHALLOW)
 	@ManyToMany
-	@JsonView({ View.FieldsOnly.class })
-	@JoinTable(indexes = { @Index(columnList = "Reference_curie"), @Index(columnList = "crossReferences_curie") })
+	@JsonView({View.FieldsOnly.class})
+	@JoinTable(indexes = {@Index(columnList = "Reference_curie"), @Index(columnList = "crossReferences_curie")})
 	@EqualsAndHashCode.Include
 	private List<CrossReference> crossReferences;
 
+
+	public static List<String> speciesPrefix;
+
+	static {
```

Comment:
@cmpich can this be moved into a Enum... we need to have a enum package that can be used for this kind of thing.